crates/rooch-framework/sources/nft/display.move

```diff
@@ -0,0 +1,43 @@
+module rooch_framework::display{
+    use std::string::String;
+    use moveos_std::simple_map;
+
+    struct Display has key, store,drop,copy {
+        sample_map: simple_map::SimpleMap<String, String>
+    }
+
+    public fun new (): Display {
+        Display {
+            sample_map: simple_map::create()
+        }
+    }
```

Comment:
I think we can make the `Display` more general. If we make the Display to an Object<Display<T>>, and generate the Object ID with `Display<T>`， The display can used to display any `struct`.

Reply:
> I think we can make the `Display` more general. If we make the Display to an Object<Display>, and generate the Object ID with `Display<T>`， The display can used to display any `struct`.

This is really nice, but such collections and nft must also carry the type Collection<T> NFT<T>

There must be a mechanism to specify that a Collection of a type can only be created once